swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from swap_meet.item import Item
```

Comment:
Do we need this import? Is there anywhere in this file that we use the `Item` identifier? Does Python know that the objects being passed into these functions are of type `Item` (or its descendants)? Does it need to know that?